docs/src/pages/layout/breakpoints/breakpoints.md

```diff
@@ -50,14 +78,12 @@ In the following demo, we change the rendered DOM element (*em*, <u>u</u>, ~~del
 
 {{"demo": "pages/layout/breakpoints/WithWidth.js"}}
 
-⚠️ `withWidth()` server-side rendering support is limited.
-
-### Render Props
+#### Render Props
 
 In some cases, you could have property name collisions using higher-order components.
 To avoid the issue, you can use the [render props](https://reactjs.org/docs/render-props.html) pattern like in the following demo.
```

Comment:
In some cases, you can experience property name collisions using higher-order components.
To avoid this, you can use the [render props](https://reactjs.org/docs/render-props.html) pattern shown in the following demo. ?